scripts/ratchet_coverage/install_cargo_llvm_cov.py

```
@@ -0,0 +1,20 @@
+# requires-python = ">=3.12"
+# dependencies = ["plumbum", "typer"]
+# ///
+from plumbum.cmd import cargo
+import typer
+
+
+def main() -> None:
+    try:
+        cargo["install", "cargo-llvm-cov"]()
+        typer.echo("cargo-llvm-cov installed successfully")
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)